docs/pages/includes/database-access/token.mdx

```diff
@@ -1,7 +1,9 @@
-The Database Service requires a valid auth token to connect to the cluster. Generate
-one by running the following command against your Teleport Auth Service and save
-it in `/tmp/token` on the node that will run the Database Service:
+{{ tokenFile="/tmp/token" }}
+
+The Database Service requires a valid auth token to join your Teleport cluster.
```

Comment:
Should this be "join token" to mirror other pages in the docs?

Reply:
good point, done: 9901f602d1c9ffcbf0d83456ebaec758c7044fae